packages/loader/container-definitions/src/deltas.ts

```diff
@@ -80,7 +80,7 @@ export interface IDeltaManagerEvents extends IEvent {
     (event: "op", listener: (message: ISequencedDocumentMessage, processingTime: number) => void);
     (event: "allSentOpsAckd", listener: () => void);
     (event: "pong" | "processTime", listener: (latency: number) => void);
-    (event: "connect", listener: (details: IConnectionDetails, opsBehind?: number) => void);
+    (event: "connect_document_success", listener: (details: IConnectionDetails, opsBehind?: number) => void);
```

Comment:
I don't think it represents the success of document connection. I think we just initiate the connection. If you see in document delta connection, this event is received on successful connection.
this.addConnectionListener("connect_document_success", (response: IConnected) => {

Reply:
I'll add a comment here -- I don't mean that "the document connection was successful" but rather literally "we received the `connect_document_success` message from the server".  It's not pretty but I believe it is accurate.

`DocumentDeltaConnection`'s `initialize()` resolves only after `connect_document_success` message is received from the server.  `initialize()` is awaited by `DocumentDeltaConnection.create()`, which is awaited by `documentService.connectToDeltaStream()`, which blocks loop exit in `DeltaManager`'s `connectCore()`, which precedes `setupNewSuccessfulConnection()` where this event gets fired.